lib/Onyx.js

```diff
@@ -293,6 +329,19 @@ function keysChanged(collectionKey, partialCollection) {
             // send the whole cached collection.
             if (isSubscribedToCollectionKey) {
                 if (subscriber.waitForCollectionCallback) {
+                    // If there is a selector, we need to reduce the collection to only contain the items with the
+                    // properties the subscriber wants
+                    if (subscriber.selector) {
+                        const newData = reduceCollectionWithSelector(cachedCollection, subscriber.selector);
+
+                        // If the data changed, give it to the subscriber
+                        if (!_.isEqual(subscriber.lastCalledWith, newData)) {
```

Comment:
Can we standardize on `deepEqual` for this since we are already using it here?

https://github.com/Expensify/react-native-onyx/blob/4efd75ce4cdc0b2e9c1ac0542565e90164572f7b/lib/OnyxCache.js#L201-L203

Reply:
Sure!